Jasmine/jasmine-standalone-3.7.1 (1)/spec/BowlingGameSpec.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+describe('BowlingGame', function(){
+  let newGame;
+  let testRoll1 = 5;
+  let testRoll2 = 2;
+  let testStrike = 10;
+  let testSpare1 = 3;
+  let testSpare2 = 7;
+
+  beforeEach(function() {
+    newGame = new BowlingGame();
+  });
+
+  describe('starts a new game', function(){
+    it('with a new game default set up', function(){
+      expect(newGame.currentFrame).toBe(1);
+      expect(newGame.strike).toBe(false);
+      expect(newGame.spare).toBe(false);
+      expect(newGame.newScorecard).toBeInstanceOf(Scorecard);
+    });
+  });
+
+  describe('roll1', function(){
+    it('confirmation of roll 1 complete', function(){
+      expect(newGame.roll1(testRoll1)).toEqual("Nice roll! Let's Roll again!");
+      expect(newGame.roll1(testRoll2)).toEqual("Nice roll! Let's Roll again!");
+    });
+  });
+
+  describe('roll2', function(){
+    it('confirmation of roll 2 complete', function(){
+      expect(newGame.roll2(testRoll1)).toEqual("Great Job! That's the end of this frame");
+      expect(newGame.roll2(testRoll2)).toEqual("Great Job! That's the end of this frame");
+    });
+  });
+
+  describe('roll3', function(){
+    it('confirmation of only being able to roll a third ball in the 10th frame', function(){
+      expect(newGame.roll3(testRoll1)).toEqual("You can only roll a third time in the 10th frame");
+      expect(newGame.roll3(testRoll2)).toEqual("You can only roll a third time in the 10th frame");
+    });
+  });
```

Comment:
I really like the confirmation - makes it clear to the user what their actions mean